tests/accounts.test.js

```diff
@@ -0,0 +1,760 @@
+    it("GET /Accounts", (done) =>{
+        chai.request(server).get("/accounts")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("array");
+            response.body.length.should.be.eq(3);
+            done();
+        });
+    });
+});
+
+describe("GET Tests Account",()=>{
+    before((done) =>{
+        const login = {
+            username: "Karla",
+	        password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            accessToken = response.body.token;
+            done();
+        });
+    });
+
+    it("GET /Accounts/ID", (done) =>{
+        chai.request(server).get("/accounts/6164db5823242f430c487fca")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(200);
+            response.body.should.be.a("object");
+            response.body.should.have.property("_id");
+            response.body.should.have.property("name");
+            response.body.should.have.property("lastname");
+            response.body.should.have.property("age");
+            response.body.should.have.property("dateBirth");
+            response.body.should.have.property("email");
+            response.body.should.have.property("username");
+            response.body.should.have.property("password");
+            response.body.should.have.property("role");
+            response.body.should.have.property("idCity");
+            done();
+        });
+    });
+
+    it("GET /Accounts/ID Not Found", (done) =>{
+        chai.request(server).get("/accounts/6164db5823242f430c487fcb")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro registro(s)");
+            done();
+        });
+    });
+
+    it("GET /Accounts/ID Bad Request", (done) =>{
+        chai.request(server).get("/accounts/6343455")
+        .auth(accessToken, { type: 'bearer' })
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+});
+
+describe("POST Tests Accounts",()=>{
+    let idAccount = null;
+
+    it("POST /Accounts Bad Request All", (done) =>{
+        const account = {};
+        chai.request(server).post("/accounts").send(account)
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+    
+    it("POST /Accounts Bad Request Name", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("POST /Accounts Bad Request Name", (done) => {
```